python/pyarrow/parquet/core.py

```diff
@@ -867,6 +915,10 @@ def _sanitize_table(table, new_schema, flavor):
     it will restore the timezone (Parquet only stores the UTC values without
     timezone), or columns with duration type will be restored from the int64
     Parquet column.
+sorting_columns : Sequence of SortingColumn, default None
+    Specify the sort order of the data being written. The writer does not sort
+    the data nor does it verify that the data is sorted. The sort order is
```

Comment:
So, user him/her self should ensure the correctness of sorting_columns?

Reply:
Yes.

Reply:
There is a danger here but I think you explain it here well enough.  I can't think of any way to be more cautionary without just being overly verbose.